Menu/Tab/index.jsx

```diff
@@ -31,6 +31,13 @@ const update = (component) => {
 export default class Tab extends Component {
   constructor () {
     super()
+
+    this.resizeListener = debounce(() => {
+      window.requestAnimationFrame
```

Comment:
You don't need to call `requestAnimationFrame` inside this. It's already done inside the `debounce`. What's going to happen now is that when the resize event happens, it schedules a callback for the next render. Once that render callback is called, it will schedule a callback for the next render. Finally in that render your update will happen.

Just do:

```
this.resizeListener = debounce(() => update(this))
```